server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -1389,6 +1390,61 @@ public void setMessagesShouldSendAReadableHtmlMessage() throws Exception {
         calmlyAwait.atMost(30, TimeUnit.SECONDS).until( () -> isHtmlMessageReceived(recipientToken));
     }
```

Comment:
Is there a test showing saving to draft is not yet implemented ?
Could be cool in my opinion...